cmd/kogito/command/converter/webhooksecret_converter.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2020 Red Hat, Inc. and/or its affiliates
```

Comment:
WDYT of  creating lots of `converter` files with single functions, would be better to have just one with all those `flag_converters` in there.

Reply:
I create independent converter/flag file for each component due to following reasions :
- Easy to debug code
- To minimize impact areas of changes
- Seperation of concern
- Help Reviewer to analyse changes
- Easy to write unit testcase

Reply:
Since you're working on it I'll believe in your experience with this code. But honestly, those functions are so simple and short that breaking this way can bring more complexity than you might think in a first glance.

Reply:
For converters I really like to keep them seperated. As a developer it help me a lot to understand code.